rpi-clone

```diff
@@ -27,18 +27,20 @@ then
     exit 1
 fi
 
-raspbian=0
-raspbian_buster=0
-if [ -f /etc/os-release ]
+rpios=0
+rpios_recent=0
+if [ -f /etc/os-release ] && [ -f /proc/device-tree/model ]
```

Comment:
As far as I see this test should check whether it's a Raspberry HW. Are you sure /proc/device-tree/model is an unique idetifier for a Raspberry HW?  `/etc/rpi-issue` exists on Raspberry OS Builds only.